.github/workflows/build.yml

```
@@ -77,7 +77,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: ["1.21","1.20"]
+        go-version: ["1.21", "1.22"]
```

We don't want to remove any support for 1.20.
```suggestion
        go-version: ["1.20", "1.21", "1.22"]
```

---

Actually, instead of modifying this, you'll need to modify the GO_SUPPORTED_VERSIONS [here](https://github.com/rook/rook/blob/release-1.13/build/makelib/golang.mk#L51).